examples/lighting-app/mtk/README.md

```diff
@@ -0,0 +1,256 @@
+#CHIP MT793X Lighting Example
+
+An example showing the use of CHIP on the MediaTek MT793X.
+
+<hr>
+
+-   [CHIP MT793X Lighting Example](#chip-mt793x-lighting-example)
+    -   [Introduction](#introduction)
+    -   [Building](#building)
+        -   [Note](#note)
+    -   [Flashing the Application](#flashing-the-application)
+    -   [Viewing Logging Output](#viewing-logging-output)
+    -   [Running the Complete Example](#running-the-complete-example)
+        -   [Notes](#notes)
+    -   [Running RPC console](#running-rpc-console)
+
+<hr>
+
+<a name="intro"></a>
+
+## Introduction
+
+The MT793X lighting example provides a baseline demonstration of a Light control
+device, built using CHIP and the MediaTek Filogic SDK. It can be controlled by a
+Chip controller over Wi-Fi network..
+
+The MT793X device can be commissioned over Bluetooth Low Energy where the device
+and the Chip controller will exchange security information with the Rendez-vous
+procedure. Network credentials are then provided to the MT793X device which will
+then join the network.
+
+The lighting example is intended to serve both as a means to explore the
+workings of CHIP as well as a template for creating real products based on the
+Silicon Labs platform.
+
+<a name="building"></a>
+
+## Building
+
+-   Download the
```

Comment:
Is this required for building? Is it in the chip/vscode image already?

We need developers to be able to compile an test on our vscode images without any additional developer steps (and explicitly no additional NDA/Agreements signed, hence some concern about instructions for downloading things to be able to compile).